File: src/main/java/jb/util/BUtils.java
package jb.util;

import java.util.ArrayList;
import java.util.List;

public class BUtils {

    public static List<int[]> getAllPermutations() {
        List<int[]> permutations = new ArrayList<>();
        for (int i = 0; i < 7; i++) {
            for (int j = i + 1; j < 8; j++) {
                permutations.add(new int[]{i, j});
            }
        }
        return permutations;
    }
}

File: src/main/java/jb/util/BUtils.kt
package jb.util

import java.util.ArrayList

class BUtils {

    companion object {
        fun getAllPermutations(): List<IntArray> {
            val permutations = ArrayList<IntArray>()
            for (i in 0..6) {
                for (j in i + 1..7) {
                    permutations.add(intArrayOf(i, j))
                }
            }
            return permutations
        }
    }
}
